workspaces/rbac/plugins/rbac/src/components/ConditionalAccess/ConditionsFormRow.tsx

```diff
@@ -505,12 +521,35 @@ export const ConditionsFormRow = ({
   };
 
   return (
-    <Box className={classes.conditionRow} data-testid="conditions-row">
+    <Box
+      sx={{
+        padding: '20px',
+        border: `1px solid ${theme.palette.border}`,
+        borderRadius: '4px',
+        backgroundColor: theme.palette.background.default,
+        '& input': {
+          color: `${theme.palette.textContrast}!important`,
+          '&:-internal-autofill-selected, &:-webkit-autofill, &:-webkit-autofill:hover, &:-webkit-autofill:focus, &:-webkit-autofill:active':
+            {
+              WebkitBoxShadow: `0 0 0px 1000px ${theme.palette.background.paper} inset`,
+              WebkitTextFillColor: `${theme.palette.textContrast}!important`,
+              caretColor: `${theme.palette.textContrast}!important`,
+            },
+        },
```

Comment:
This tries to remove the auto fill color of an input field? Is this necessary?

Reply:
Yeah, so this is done for the dynamic fields generated using rjsf and the original auto fill color doesnt match the UX so this was a workaround that @ciiay came up with as at that time nothing else seemed to work. But we can surely think about if this can be improved.
How it should look like
<img width="1361" alt="Screenshot 2024-11-19 at 6 07 41 PM" src="https://github.com/user-attachments/assets/04db398b-5da5-4782-908a-a723495347c0">

How it looks without above changes

https://github.com/user-attachments/assets/e1522b40-dfbe-47c3-b348-2e7edb04d33d